app/routes/__index/dns-records/index.tsx

```diff
@@ -97,9 +97,11 @@ export default function DnsRecordsIndexRoute() {
           DNS Records
         </Heading>
         <Text mb="4" mt="2">
-          Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has
-          been the industry's standard dummy text ever since the 1500s, when an unknown printer took
-          a galley of type and scrambled it to make a type specimen book.
+          This table shows all of your DNS records that you have created. Once you create a new
+          record, you will immediately see that record on the table. However, it will take some time
+          to go into effect as your new domain needs to be spread in DNS servers around the globe.
+          Expiration date is initially set to 6 months later from creation date. You can renew
+          expiration date.
```

Comment:
```suggestion
          Expiration date is initially set to 6 months later from creation date. You can renew
          the record by clicking the renew button next to the expiration date. This will set the
          new expiration date to 6 months from now.
```

Reply:
@humphd Unrelated but I wonder if we should change when the renew button is disabled? We enable the renew button when the expiration date is less than 6 months from now. But since the expiration date is set to 6 months from now, I am able to immediately renew the dns record after creating it